docs/aro/ocm.md

```diff
@@ -0,0 +1,37 @@
+# Registering an ARO cluster to OpenShift Cluster Manager 
+
+ARO clusters do not come connected to OpenShift Cluster Manager by default,
+because Azure would like customers to specifically opt-in to connections / data
```

Comment:
we can say something "for privacy and reasons ARO clusters do not..." to show it as a mutual decision between azure/redhat to ensure the customer owns their privacy.